index.html

```diff
@@ -1 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title> José Hernández-López's Portfolio</title>
+    <meta name="viewpoint" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="A public JavaScript portfolio">
+</head>
+    <h1>José Hernández-López</h1>
+    <h2>About</h2>
```

Comment:
` Each group of related content, such as 'About', 'Experience', and 'Connect', should be contained in distinct <section> elements`